config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -526,8 +527,8 @@ periodics:
       - --scenario=kubernetes_e2e
       - --
       - --cluster=kubemark-100pods
-      - --env-file=jobs/env/ci-kubernetes-e2e-kubemark-common.env
-      - --env-file=jobs/env/ci-kubernetes-kubemark-high-density-100-gce.env
+      # Reduce logs verbosity
+      - --env=TEST_CLUSTER_LOG_LEVEL=--v=2
```

Comment:
Can you add a TODO for me to get rid of that (or unify with other jobs where we reduce verbosity)?

Reply:
if it's not needed I can get rid of it? (Looks like it only references kubemark-common which doesn't have the verbosity option)

Reply:
Log verbosity has impact on performance. So let's not mix such changes with biggest refactorings.
Please add a TODO for me to validate if we can safely do that.